server/test/com/cloud/network/firewall/FirewallManagerTest.java

```diff
@@ -71,6 +79,7 @@
 //        super.setUp();
 //    }
 
+    @Ignore("Requires database to be set up")
```

Comment:
Because it says: "Requires database to be set up"

So I'm assuming that the person who wrote it either did not know how to mock the DB layer or the ACS is just to messed up to a point where mocking DB stuff is almost impossible.